asab/api/doc.py

```diff
@@ -30,11 +30,13 @@ def __init__(self, api_service, app, web_container, config_section_name="asab:do
 		self.WebContainer.WebApp.router.add_get("/asab/v1/openapi", self.openapi)
 
 		self.AuthorizationUrl = asab.Config.get(
-			config_section_name, "authorizationUrl", fallback=None
+			config_section_name, "authorization_url", fallback=None
 		)
-		self.TokenUrl = asab.Config.get(config_section_name, "tokenUrl", fallback=None)
+		self.TokenUrl = asab.Config.get(config_section_name, "token_url", fallback=None)
 		self.Scopes = asab.Config.get(config_section_name, "scopes", fallback=None)
-
+		self.GlobalSecurity = asab.Config.getboolean(config_section_name, "global_security")
+		self.ClientId = asab.Config.get(config_section_name, "client_id")
```

Comment:
sorry, i was wrong about the client ID, there is a different way to configure it. please delete this option for now, i'll make a separate issue for this